02.calendar/calendar.rb

```diff
@@ -10,13 +10,14 @@
 
 puts "日 月 火 水 木 金 土"
 
-first_wday = Date.new(year, month, 1).wday
-last_day = Date.new(year, month, -1).day
-print "   " * first_wday
+first_date = Date.new(year, month, 1).day
```

Comment:
dayはDateオブジェクトではなく、整数値（Integer）です。

https://docs.ruby-lang.org/ja/latest/method/Date/i/day.html

なので、僕としては以下のように書く方が望ましいと考えていますー。

```suggestion
first_date = Date.new(year, month, 1)
```